src/keyfile.c

```diff
@@ -1073,17 +1073,27 @@ static void load_dialog_prefs(GKeyFile *config)
 	if (!EMPTY(tmp_string2))
 	{
 	#ifdef G_OS_WIN32
-		tmp_string = g_strconcat(GEANY_DEFAULT_TOOLS_PRINTCMD, " \"%f\"", NULL);
+		tmp_string = g_strconcat(GEANY_DEFAULT_TOOLS_PRINTCMD, " \"%d\\%f\"", NULL);
```

Comment:
Can't we use `G_DIR_SEPARATOR_S` here as well and so save the `#ifdef`?

Reply:
It doesn't use the same quoting, so if there's a reason for this I guess not.

Reply:
Ok, then better let's not break even more things before a release :).